### DISCUSSION TITLE
Temporary Maintenance Hiatus Due to Illness

### DISCUSSION THREAD

**Discussion by kagemomiji** [kagemomiji/airsonic-advanced#562] [Announcements]
Hello everyone,

I hope this message finds you well. Unfortunately, my family and I have contracted COVID-19 and I will need to take a break from maintenance activities for the next two weeks (may be until August 16th) to focus on our recovery.

I apologize for any inconvenience this may cause and appreciate your understanding and patience during this time. I will resume my activities as soon as I am able.

Thank you for your support.

**Comment by masterwishx** [comment]
Good recovery to you and your family, Thanks for your hard work for this project.

**Comment by uSlackr** [comment]
Hope you all heal quickly. But it is an  exhausting illness


\\Greg
________________________________

**Comment by kagemomiji** [comment]
Hello everyone,

I’m happy to share that my family and I have fully recovered from Covid-19. As a result, I am now able to resume maintenance activities.

Thank you all for your understanding and patience during our recovery period. I’m looking forward to getting back to work and continuing my progress.

**Reply by Dikakus** [reply]
Hi, @kagemomiji. I'm glad to hear that you and your family have now recovered.

And thank you for continuing to maintain this app.
